src/encoder/iiif_encoder.rs

```diff
@@ -20,11 +22,11 @@ pub struct IiifEncoder {
 }
 
 impl IiifEncoder {
-    pub fn new(destination: PathBuf, size: Vec2d) -> Result<Self, ZoomError> {
+    pub fn new(destination: PathBuf, size: Vec2d, quality: u8) -> Result<Self, ZoomError> {
         let _ = std::fs::remove_file(&destination);
         debug!("Creating IIIF  directory at {:?}", &destination);
         std::fs::create_dir(&destination)?;
-        let tile_saver = IIIFTileSaver { root_path: destination.clone() };
+        let tile_saver = IIIFTileSaver { root_path: destination.clone(), quality: quality };
```

Comment:
```suggestion
        let tile_saver = IIIFTileSaver { root_path: destination.clone(), quality };
```

Reply:
Thanks - I thought that there must be a more elegant way of doing it!